### ISSUE TITLE
There is no way to give feedback and/or rating on exercises

### ISSUE THREAD

**Issue by s-badran** [exercism/website#1727]
I come across some exercises that are not correctly worded, tested, or even assigned correct difficulty. I think a mechanism for feedback on each exercise is very important for a community-based platform such as exercism.

E.g. Difficulty level is relative from person to another. The author of the question could see it very easy but to the majority of the crowd its a hard question! I think it makes sense to allow users to **rate the difficulty of questions** (similar to what Leedcode has).

E.g. Some exercises are way too wordy or unnecessarily complicated for the concept being taught! It helps for others to **rate the value of the question** (relative to the concept) and let them decide if they want to invest time on it or not.

**Comment by iHiD** [comment]
Thanks @salouri.

I agree with your points. In the interim, you can open issues on the various GitHub repos where you find problems (e.g. https://github.com/exercism/elixir) and the maintainers will fix them.
